apis/core.oam.dev/v1alpha2/environment_types.go

```diff
@@ -29,13 +29,19 @@ type EnvironmentSpec struct {
 	Namespace string `json:"namespace"`
 
 	// +kubebuilder:validation:Type=string
+	// +optional
 	Email string `json:"email,omitempty"`
 
 	// +kubebuilder:validation:Type=string
+	// +optional
 	Domain string `json:"domain,omitempty"`
 
 	// +kubebuilder:validation:Type=string
-	Issuer string `json:"issuer"`
+	// +optional
+	Issuer string `json:"issuer,omitempty"`
+
+	// +optional
+	CapabilityList []string `json:"capabilityList,omitempty"`
```

Comment:
Why do we need this field?

Reply:
According to [this suggestion](https://github.com/oam-dev/kubevela/pull/999#discussion_r569928291).